pkg/cmd/server/start/start_master.go

```diff
@@ -755,7 +755,7 @@ func startControllers(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) erro
 	oc.RunSDNController()
 
 	// initializes quota docs used by admission
-	oc.RunResourceQuotaManager(nil)
+	oc.RunResourceQuotaManager(controllerManagerOptions)
```

Comment:
Cleanup this method to prevent nils and respect the config.  I don't see a reason to ever set it without these options.